select_22.md

```diff
@@ -0,0 +1,7 @@
+### 22. Lấy danh sách user đã comment trong 2 blog mới nhất
+```mysql
+SELECT DISTINCT `user`.* FROM `user` WHERE id IN
+   (SELECT `comment`.user_id from `comment`
+   INNER JOIN (SELECT user_id from blog ORDER BY created_at DESC) blog_id_user
```

Comment:
updated_at, thiếu limit trong 2 blog mới nhất. 
(trong where in ko dùng limit được nên dùng cách khác)